pkg/quota/v1/evaluator/core/pods.go

```diff
@@ -155,14 +155,11 @@ func (p *podEvaluator) GroupResource() schema.GroupResource {
 
 // Handles returns true if the evaluator should handle the specified attributes.
 func (p *podEvaluator) Handles(a admission.Attributes) bool {
-	op := a.GetOperation()
```

Comment:
I'm concerned this won't actually work due to:

https://github.com/kubernetes/kubernetes/blob/5b1fffa3e40b812e81ede244f671c90e3428e2ec/pkg/kubeapiserver/admission/exclusion/resources.go#L30

Can we add a test that proves me wrong? (adding /resize to that include list if needed?)

Reply:
Actually... per our offline conversation.. I think this included list is only for ValidatingAdmissionPolicy. We still need to add resize to the list, but quota should work.